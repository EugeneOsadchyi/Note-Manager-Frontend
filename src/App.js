import React from 'react';

const App = () => (
  <div>
    <h1>
      NoteManager
    </h1>
  </div>
);

export default App;
