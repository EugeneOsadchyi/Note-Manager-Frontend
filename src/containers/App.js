import React from 'react';

const App = () => (
  <div>
    <h1>
      Note Manager
    </h1>
  </div>
);

export default App;
