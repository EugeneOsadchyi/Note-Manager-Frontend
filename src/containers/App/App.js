import React from 'react';

import './App.css';

import Layout from '../../hoc/Layout';
import Explorer from '../Explorer';

import Modal from '../../components/UI/Modal';
import Input from '../../components/UI/Input';
import Button from '../../components/UI/Button';

const App = () => (
  <div className="App">
    <Layout>
      <Explorer />
      <Modal show>
        <h2>
          New Directory
        </h2>
        <Input placeholder="Name" />
        <Button clicked={() => alert('Test')}>
          Cancel
        </Button>
        <Button btnType="Success" clicked={() => alert('Test')}>
          Save
        </Button>
      </Modal>
    </Layout>
  </div>
);

export default App;
