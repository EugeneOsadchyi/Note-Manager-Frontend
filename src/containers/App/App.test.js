import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import App from './App';

describe('App', () => {
  const app = shallow(<App />);

  it('renders properly', () => {
    expect(shallowToJson(app)).toMatchSnapshot();
  });

  // it('contains the connected explorer component', () => {
  //   expect(app.find('Connect(Explorer)').exists()).toBe(true);
  // });
});
