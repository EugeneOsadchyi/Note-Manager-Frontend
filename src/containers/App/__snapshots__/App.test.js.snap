// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders properly 1`] = `
<div
  className="App"
>
  <Layout>
    <Connect(Component) />
    <Modal
      show={true}
    >
      <h2>
        New Directory
      </h2>
      <Input
        placeholder="Name"
      />
      <Button
        btnType={null}
        clicked={[Function]}
        disabled={false}
      >
        Cancel
      </Button>
      <Button
        btnType="Success"
        clicked={[Function]}
        disabled={false}
      >
        Save
      </Button>
    </Modal>
  </Layout>
</div>
`;
